scripts/install_xcpng.py

```diff
@@ -30,21 +30,26 @@ def pxe_address():
     except ImportError:
         raise Exception('No address for the PXE server found in data.py (`PXE_CONFIG_SERVER`)')
 
-def generate_boot_conf(directory, installer):
+def generate_boot_conf(directory, installer, action):
+    if action == 'restore':
+        rt = 'rt=1'
+    else:
+        rt = ''
```

Comment:
```suggestion
    rt = 'rt=1' if action == 'restore' else ''
```
It's not blocking for the merge but it is more concise and more inline with the python standard coding style.